frontend/components/Seo/index.tsx

```diff
@@ -0,0 +1,35 @@
+import Head from 'next/head'
+
```

Comment:
Don't need these returns here and in `frontend/pages/index.tsx `

I pass but try remember to get rid of them later aha XD

Reply:
That went without my knowledge 😂